src/uu/ls/src/ls.rs

```diff
@@ -2564,3 +2482,92 @@ fn get_security_context(config: &Config, p_buf: &Path, must_dereference: bool) -
         substitute_string
     }
 }
+
+#[cfg(unix)]
+fn calculate_padding_collection(
+    items: &[PathData],
+    config: &Config,
+    out: &mut BufWriter<Stdout>,
+) -> PaddingCollection {
+    let (
+        mut longest_inode_len,
+        mut longest_link_count_len,
+        mut longest_uname_len,
+        mut longest_group_len,
+        mut longest_context_len,
+        mut longest_size_len,
+        mut longest_major_len,
+        mut longest_minor_len,
+    ) = (1, 1, 1, 1, 1, 1, 1, 1);
```

Comment:
If we're refactoring this anyway, we could create the padding at the start and then update it, instead of creating these intermediate variables and rename the fields of the padding collections from `longest_inode_len` to just `inode`.